tools.js

```diff
@@ -59,6 +59,45 @@ module.exports = (function () {
 		timid: {name:"Timid", plus:'spe', minus:'atk'}
 	};
 
+	function tryRequire (path) {
```

Comment:
In theory our convention is `tryRequire(`.

We require a space after the `function` keyword, but no space between the function name and `(`.